src/components/layouts/EditForm/index.tsx

```diff
@@ -0,0 +1,94 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+import Button from '../../ui/Button';
+import EditIcon from '../../ui/EditIcon';
+import convertFromPixelsToRem from '../../../settings/__convertToCssRemUnit';
+
+interface Props {
+  handleSubmit: () => void;
+  loading?: boolean;
+  imageLink?: string;
+  displayHeader?: boolean;
+  imageName?: string;
+  submitButtonName: string;
+  children: React.ReactNode;
+}
+
+const MobileForm = (props: Props) => {
```

Comment:
I'm kinda confused, the file name is `EditForm` and its default export is `MobileForm`. Please the form name and file name should be the same.
This should be a regular function and not an arrow function.

Reply:
this was for testing purpose. I forgot to take it out.

Reply:
resolved.